DTML.EduBot/Dialogs/TranslateDialog.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Luis.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace DTML.EduBot.Dialogs
+{
+    public partial class HelpDialog : QnaLuisDialog<object>
+    {
+        [LuisIntent("Translate")]
```

Comment:
Why Translate intent? Right now we have a translate API in place. Plus, if the user needs help, it should be handled by help intent, followed by yes/no choice, and then we call translate API.